site/gatsby-site/content/research/5-response/index.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: "Defining an “AI Incident Response”"
+metaTitle: "Defining an “AI Incident Response”"
+metaDescription: "What is an AI incident response?"
+slug: "/research/5-response"
+---
+
+An **AI Incident Response** refers to a **public official response** to an **[incident](/research/1-criteria)** in the AI Incident Database (AIID) **from an [entity](/entities)** (i.e. company, organization, individual) allegedly **responsible for** developing or deploying the AI or AI system involved in said incident.
+
+An AI Incident Response typically contains reasonably adequate information about the incident regarding:
+
+1. What happened,
+2. Why the incident happened,
+3. What the entity has done or is going to do to
+   <ol style={{ marginLeft: '2em' }} type="a">
+      <li>remediate the incident, and</li>
+      <li>prevent future incidents from recurring.</li>
+   </ol>
+
+## Submitting Incident Responses
+
+The AI Incident Database is designed to provide information, structure, and perspectives on AI incidents. If you are listed as an entity for one or more incidents, we would very much like to work with you to provide your perspective. You can either [submit a response directly](#todo) to the database, or [contact us](/contact) to discuss submitting a response.
```

Comment:
We should change `#todo` link to `/app/submit`

Reply:
Thanks, fixed.